Ginger/Ginger/AutomatePageLib/AddActionMenu/ActionsFactory.cs

```diff
@@ -269,10 +270,29 @@ public static void AddActivitiesFromSRHandler(List<Activity> sharedActivitiesToA
 
             if (parentGroup != null)
             {
+
                 foreach (Activity sharedActivity in sharedActivitiesToAdd)
                 {
+                    eUserMsgSelection userSelection = eUserMsgSelection.None;
                     Activity activityIns = (Activity)sharedActivity.CreateInstance(true);
                     activityIns.Active = true;
+                    //map activities target application to BF if missing in BF
```

Comment:
This code to map activities TA to BF is duplicated in 2 places. Can we move it to common method and call this method ?

Reply:
Created function to map TA to BF for duplicate code